src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.PackTool.targets

```diff
@@ -0,0 +1,33 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.PackTool.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="PackTool" DependsOnTargets="Publish" Condition=" '$(IsPackedAsTool)' == 'true' ">
```

Comment:
I skip _IsPackedAsTool_ at target level because this property is set in csproj, and SDK prop _TargetsForTfmSpecificContentInPackage_ will get evaluated before reading csproj

Reply:
Could this property be just named `PackAsTool`? seems a little more direct..

Reply:
oops, ya exactly pack as tool

Reply:
fixed